format/src/main/scala/org/scalafmt/internal/TreePrinter.scala

```diff
@@ -250,7 +250,8 @@ object TreePrinter {
           case t: Term.Eta =>
             print(t.expr) + space + `wildcard`
           case t: Term.ApplyUnary =>
-            print(t.op) + SimpleExpr.wrap(t.arg)
+            val group = TreeSyntacticGroup(t)
```

Comment:
Can we do `PrefixExpr(t.op.value)` instead? I think that would be consistent with how it's done in other cases

Reply:
I would prefer to go the other way around: add more of those.

Reply:
Will this every be anything else than `PrefixExpr(t.op.value)`?

Reply:
We already have a function to compute this.

Reply:
I agree it would make sense to consolidate TreeSyntacticGroups with the custom SimpleExpr/... in TreePrinter.

Can we use `PrefixExpr(t.op.value)` here for consistency with the rest of TreePrinter and then follow up in another PR to migrate everything to TreeSyntacticGroups (which we might want to rename)? Consistency makes the code easier to follow, when I saw this change here I thought something special was happening that required extra attention.